extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -136,10 +136,15 @@ function checkAdBlocking(detection) {
 	);
 }
 
+function passSlotEvent(slotName, eventName) {
```

Comment:
We don't use this method. Do you see a value in keeping it in code?

Reply:
It's not used now, but on mobile-wiki someone added this method:
https://github.com/Wikia/mobile-wiki/blob/c12cf9a36abd9f842a3fe94542f70d4961658a98/app/modules/ads.js#L192
So in the future it might be good to have working function to pass events to AdEngine 3, when he will be back to this task.